### PR TITLE
Add contribution guidelines

### PR DESCRIPTION
**Pull request by attwad**
Pretty standard stuff in there, should raise our community profile status:
https://github.com/globaldothealth/list/community

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Contributing to the Global.Health list
+
+First of all, thank you for willing to contribute some of your time to the GLobal.Health list, you rock! :guitar:
```
Comment (axmb) [inline CONTRIBUTING.md:3]: Perhaps s/willing/choosing

Comment (axmb) [inline CONTRIBUTING.md:3]: lower-case the L

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by our [Code of Conduct](https://github.com/globaldothealth/list/blob/main/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
+
+## What can you contribute to?
+
+The Global.Health list is a project with multiple services in it, depending on your area of expertise you might want to start with looking for [issues](https://github.com/globaldothealth/list/issues) that are relevant to your skills.
+
+The stack is mostly Node.JS/Typescript MongoDB React based, with some Python in it.
```
Comment (axmb) [inline CONTRIBUTING.md:15]: MERN, as the kids call it these days.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+- [Infrastructure](https://github.com/globaldothealth/list/labels/Infrastructure)
+- [Parsers for automated ingestion](https://github.com/globaldothealth/list/labels/importer)
+- [Curator portal UI/React](https://github.com/globaldothealth/list/issues?q=is%3Aopen+is%3Aissue+label%3A%22Curator+Portal%22)
+- [Documentation](https://github.com/globaldothealth/list/labels/Documentation) because there is never enough of it.
+
+You can check out the [list of all labels](https://github.com/globaldothealth/list/labels) as well to help you figure out where to start.
+
+## Report issues
+
+A very good way to contribute is to report issues, either found when working on a PR or when simply using the Global.Health list, we have issue templates to help you report feature requests or bugs, please use them when [reporting an issue](https://github.com/globaldothealth/list/issues/new/choose).
```
Comment (axmb) [inline CONTRIBUTING.md:39]: nit: period here (`[...] or when simply using the Global.Health list. We have issue templates [...]`).

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+The documentation for devs can be found in the [`/dev`](https://github.com/globaldothealth/list/blob/main/dev/README.md) directory.
+
+It should contain all the necessary information to run a full stack locally and get you started on your first PR.
+
+We use the [Git Feature Branch Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) for development. You should always have a working `main` branch and features being developed in parallel branches then merged into `main`.
+
+## Testing
+
+We have numerous [github actions](https://github.com/globaldothealth/list/actions) that run on every PR and push to the `main` branch. Make sure your PR actions are green before requesting a review. We also protect the `main` branch so reviews are required, you will not get an approval if the actions are not passing so please, and and fix tests whenever possible.
```
Comment (axmb) [inline CONTRIBUTING.md:51]: > please, and and

Might have dropped a word here?

**Comment by attwad** [comment]
> Nice, thanks! Should I take a swing at the PR template? ;-)

may your wish be granted

**Pull request merged**